rust/libceed/src/lib.rs

```diff
@@ -54,12 +54,18 @@ pub mod operator;
 pub mod qfunction;
 pub mod vector;
 
+// -----------------------------------------------------------------------------
+// Typedef for scalar
+// -----------------------------------------------------------------------------
+pub type Scalar = bind_ceed::CeedScalar;
+
 // -----------------------------------------------------------------------------
 // Constants for library
 // -----------------------------------------------------------------------------
 const MAX_BUFFER_LENGTH: u64 = 4096;
 pub const MAX_QFUNCTION_FIELDS: usize = 16;
 pub const CEED_STRIDES_BACKEND: [i32; 3] = [0; 3];
+pub const EPSILON: crate::Scalar = bind_ceed::CEED_EPSILON as crate::Scalar;
```

Comment:
This was pretty painless - I just propagated this typedef from libCEED all of the way through to our Rust examples